packages/paste-website/src/pages/components/display-heading/index.mdx

```diff
@@ -0,0 +1,228 @@
+</LivePreview>
+
+### Display Heading as h2 with heading20 variant
+
+<LivePreview scope={{DisplayHeading}} language="jsx">
+  {`<DisplayHeading as="h2" variant="displayHeading20">
+  Display Heading two
+</DisplayHeading>`}
+</LivePreview>
+
+### Heading as h3 with displayHeading30 variant
+
+<LivePreview scope={{DisplayHeading}} language="jsx">
+  {`<DisplayHeading as="h2" variant="displayHeading30">
+  Display Heading three
+</DisplayHeading>`}
+</LivePreview>
+
+### Display Heading with no bottom margin
+
+<LivePreview scope={{DisplayHeading, Card}} language="jsx">
+  {`<Card>
+  <DisplayHeading as="h2" marginBottom="space0" variant="displayHeading10">
+    Display Heading with no margin
+  </DisplayHeading>
+</Card>`}
+</LivePreview>
+
+## Composition Notes
+
+### When to use Display Heading
+
+<DoDont>
+  <Do title="Do" body="Use sentence case in Display Headings (except for a product name)." center>
+    <DisplayHeading as="h2" variant="heading10">
+      This is sentence case
+    </DisplayHeading>
+  </Do>
+  <Dont title="Don't" body="Don’t use title case." center>
+    <DisplayHeading as="h2" variant="heading10">
+      This Is Title Case
+    </DisplayHeading>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use Display Headings to connect with the brand or grab users' attention." />
+  <Dont title="Don't" body="Don’t use Display Headings in place of the regular Heading component." />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Be intentional about the value you pass to the `as` prop." />
+  <Dont title="Don't" body="Don’t use multiple h1 tags on a page." />
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/display-heading - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {DisplayHeading} from '@twilio-paste/core/display-heading';
+
+const Component = () => (
+  <DisplayHeading as="h2" variant="displayHeading20">
+    Display Heading
+  </DisplayHeading>
+);
+```
+
+#### Props
+
+| Prop          | Type                           | Description                                                                               | Default           |
+| ------------- | ------------------------------ | ----------------------------------------------------------------------------------------- | ----------------- |
+| as            | `asTags`                       | 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div', 'span'                                         | null              |
+| id?           | `string`                       |                                                                                           | null              |
+| marginBottom? | `oneOf(['space0'])`            | Currently we only allow `space0` to remove bottom margin                                  | null              |
+| variant       | `displayHeadingVariants`       | 'heading10', 'heading20', 'heading30'                                                     | null              |
```

Comment:
issue: double check these props. Variants don't match as this might be a copy paste from Heading.

Reply:
thanks for catching this, fixed